suite-native/module-dev-utils/src/components/LocalFirstRelaySettings.tsx

```diff
@@ -0,0 +1,112 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+    DEFAULT_LOCAL_FIRST_STORAGE_RELAY_URL,
+    disposeAllLocalFirstStorageThunk,
+    labelingActions,
+    selectIsLocalFirstStorageEnabled,
+    selectLocalFirstStorageRelayUrl,
+} from '@suite-common/local-first-storage';
+import { yup } from '@suite-common/validators';
+import { Button, Card, CheckBox, HStack, Text, VStack } from '@suite-native/atoms';
+import { Form, TextInputField, useForm } from '@suite-native/forms';
+import { initNativeLocalFirstStorageThunk } from '@suite-native/local-first-storage';
+import { useToast } from '@suite-native/toasts';
+
+export const LocalFirstRelaySettings = () => {
+    const localFirstStorageRelayUrl = useSelector(selectLocalFirstStorageRelayUrl);
+    const isLocalFirstStorageEnabled = useSelector(selectIsLocalFirstStorageEnabled);
+    const dispatch = useDispatch();
+    const { showToast } = useToast();
+
+    const form = useForm<{ localFirstStorageRelayUrl: string }>({
+        defaultValues: {
+            localFirstStorageRelayUrl: localFirstStorageRelayUrl ?? '',
```

Comment:
Why not to pass `DEFAULT_LOCAL_FIRST_STORAGE_RELAY_URL` as default here directly?
```suggestion
            localFirstStorageRelayUrl: localFirstStorageRelayUrl ?? DEFAULT_LOCAL_FIRST_STORAGE_RELAY_URL,
```